netlify.toml

```diff
@@ -1,8 +1,8 @@
 [dev]
-  command = "npm run dev"
+  command = "bun run dev"
```

Comment:
Yay bun!

Reply:
I realized I had created this site before I knew about bun lol